website/content/en/docs/helm/quickstart.md

```diff
@@ -231,83 +230,113 @@ using a bit of setup from other `operator-sdk` commands. Check out the OLM integ
+    ...
+    replicaCount: 3
+    resources: {}
+    securityContext: {}
+    serviceAccount:
+      annotations: {}
+      create: true
+      name: ""
+    tolerations: []
+```
 
-$ kubectl apply -f deploy/crds/example.com_v1alpha1_nginx_cr.yaml
+And apply the change:
+
+```sh
+$ kubectl apply -f config/samples/example_v1alpha1_nginx.yaml -n nginx-operator-system
 ```
 
 Confirm that the operator changes the deployment size:
 
 ```sh
-$ kubectl get deployment
-NAME                                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-example-nginx-b9phnoz9spckcrua7ihrbkrt1        3         3         3            3           1m
+$ kubectl get deployment -n nginx-operator-system
+NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
+nginx-operator-controller-manager   1/1     1            1           9m16s
+nginx-sample                        3/3     3            3           7m29s
 ```
 
 Check that the service port is set to the default (`80`):
 
 ```sh
-$ kubectl get service
-NAME                                      TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)  AGE
-example-nginx-b9phnoz9spckcrua7ihrbkrt1   ClusterIP   10.96.26.3   <none>        80/TCP   1m
+$ kubectl get service  -n nginx-operator-system
+NAME                                                TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+nginx-operator-controller-manager-metrics-service   ClusterIP   10.99.175.79    <none>        8443/TCP            9m42s
+nginx-operator-metrics                              ClusterIP   10.107.164.77   <none>        8383/TCP,8686/TCP   9m13s
+nginx-sample                                        ClusterIP   10.96.152.76    <none>        80/TCP              7m54s
+```
+
+### Troubleshooting
+
+Use the following command to check the operator logs.
+
+```sh
+$ kubectl logs deployment.apps/nginx-operator-controller-manager  -n nginx-operator-system -c manager
+```
+
+Use the following command to check the CR status and events.
+
+```sh
+$ kubectl describe Nginx -n nginx-operator-system 
 ```
 
 ### Cleanup
 
 Clean up the resources:
 
 ```sh
-kubectl delete -f deploy/crds/example.com_v1alpha1_nginx_cr.yaml
-kubectl delete -f deploy/operator.yaml
-kubectl delete -f deploy/role_binding.yaml
-kubectl delete -f deploy/role.yaml
-kubectl delete -f deploy/service_account.yaml
-kubectl delete -f deploy/crds/example.com_nginxes_crd.yaml
+$ kubectl delete -f config/samples/example_v1alpha1_nginx.yaml -n nginx-operator-system
+$ make undeploy
 ```
-**NOTE** Additional CR/CRD's can be added to the project by running, for example, the command :`operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=AppService`
+**NOTE** Additional CR/CRD's can be added to the project by running, for example, the command :`operator-sdk create api --group=example --version=v1alpha1 --kind=AppService`
 For more information, refer [cli][addcli] doc.
```

Comment:
This seems like a strange place for this sentence to show up. Maybe we need a larger re-organization of the Helm docs. :thinking:

Reply:
I really think that we need a large organization for all types. But we can do it as follow as well.